src/driver/include/driver/DefaultDriver.hpp

```diff
@@ -28,8 +28,24 @@ namespace genny::driver {
  * Basic workload driver that spins up one thread per actor.
  */
 class DefaultDriver {
-
 public:
+    enum class RunMode {
+        kNormal = 0,
```

Comment:
[style] Is there any value to specify `= 0` here? Unlike the `OutcomeCode` enum we aren't trying to convert the `RunMode` enum cases to their underlying integer value.

Reply:
Nah. Removed.